apps/services/sessions/infra/sessions.ts

```diff
@@ -99,3 +116,37 @@ export const workerSetup = (): ServiceBuilder<'services-sessions-worker'> =>
       },
       REDIS_USE_SSL: 'true',
     })
+
+export const geoipSetup =
+  (): ServiceBuilder<'services-sessions-geoip-worker'> =>
+    service('services-sessions-geoip-worker')
+      .image(imageName)
+      .namespace(namespace)
+      .redis()
+      .serviceAccount('sessions-geoip')
+      .command('node')
+      .replicaCount({ min: 1, max: 1, default: 1 })
+      .args(
+        './node_modules/geoip-lite/scripts/updatedb.js',
+        'license_key=$(GEOIP_LICENSE_KEY)',
+      )
+      .resources({
+        limits: {
+          cpu: '200m',
+          memory: '256Mi',
+        },
+        requests: {
+          cpu: '10m',
+          memory: '128Mi',
```

Comment:
Is the delta between the cpu limits not quite high? This could possibly cause issues with over provisioning when many nodes are starting up such as in deployments or large scale ups

Reply:
you think that also affects cronjobs?

Reply:
Hmm, the limits do affect cron jobs, maybe we should just see how much this actually requires and tune it after the fact? I still think 10m cpu is quite small. Maybe put this up to 150? That allows the container to use 0.15% of a CPU core. I do however suspect it will require more but that can just be tuned later